src/image-carousel/image-carousel.style.tsx

```diff
@@ -0,0 +1,196 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { ClickableIcon } from "../shared/clickable-icon";
+
+export const CloseButton = styled(ClickableIcon)`
+    padding: 0;
+    order: -1; // show button on the left of the header
+    border-radius: 100%;
+    background: white;
+    color: ${Color.Primary};
+    height: 2.5rem;
+    width: 2.5rem;
+    position: absolute;
+    top: 3rem;
+    right: 3rem;
+    z-index: 5;
+    ${MediaQuery.MaxWidth.tablet} {
+        top: 2rem;
+        right: 2rem;
+    }
```

Comment:
have confirmed with UX that tablet-specific media query is not required, tablet viewports should follow desktop

Reply:
this is for case mobile in horizontal mode